rllib/core/optim/tests/test_rl_optimizer_torch.py

```diff
@@ -97,33 +96,46 @@ def __init__(self, config: FCConfig) -> None:
         super().__init__(config)
         self.policy = FCNet(config)
 
-    @override(RLModule)
+    def input_specs(self) -> ModelSpec:
```

Comment:
See comment below on synching with master. All changes in this file seem to be outdated.